dump_javascript_urls.mjs

```diff
@@ -139,3 +139,7 @@ function FormatNode(node, noWrap = false) {
 
 	return str;
 }
+
+function FormatFinalUrl(str) {
+	return str.trim().replace(/\/$/, "").replace(/\/\#/, "#").replace(/\/\?/, "?");
```

Comment:
`trim()` may be unnecessary, but I put it there for good measure.
`replace(/\/$/, "")` removes any trailing slashes.
`replace(/\/\#/, "#")` replaces `/#` with just `#`.
`replace(/\/\?/, "?")` replaces `/?` with just `?`.